tests/testsearch.py

```diff
@@ -390,6 +390,86 @@ def testLargePagination(self):
             ok = False
         self.assertTrue(ok)
 
+    def testChemSearch(self):
+        """Test the chemical attribute search using both the operator and
+        fluent syntaxes. """
+        q1 = attrs.drugbank_info.brand_names.contains_phrase("Tylenol")  # 111 results 19/06/23
+        result = list(q1())
+        ok = len(result) > 0
+        self.assertTrue(ok)
+        ok2 = "1TYL" in result
+        logger.info("Chemical Search Operator Syntax: result length: (%d), ok: (%r), ok2: (%r)", len(result), ok, ok2)
+
+        result = TextQuery("Hemoglobin")\
+            .and_("chem_comp.name", CHEMICAL_ATTRIBUTE_SEARCH_SERVICE).contains_phrase("adenine")
+        # result = set(result("assembly"))
+        q1 = TextQuery("Hemoglobin")
+        q2 = attrs.chem_comp.name.contains_phrase("adenine")
+        result2 = q1 & q2
+        # result2 = set(result2("assembly"))
+        ok = result == result2  # check why this doesn't work tomorrow
+        logger.info("result of first query:")
+        logger.info(result)
+        logger.info("result of second query:")
+        logger.info(result2)
+        self.assertTrue(ok)
+        resultL = list(result2())
+        ok2 = "6FJH" in resultL
+        self.assertTrue(ok2)
+        logger.info("Chemical Search Fluent Syntax: result length: (%d), ok: (%r), ok2: (%r)", len(resultL), ok, ok2)
+
+    def testMismatch(self):
```

Comment:
Perhaps add info on what this test does (similar to the other). 

Is there a test for mismatched type and attribute?

Reply:
Yes - this is what testMismatch does.

Reply:
@santiblaumann Thanks. I see the `rcsb_chem_comp.name` and `STRUCTURE_ATTRIBUTE_SEARCH_SERVICE` combination.